client/src/components/Monument/Map/Map.js

```diff
@@ -11,7 +11,7 @@ export default class Map extends React.Component {
             <div className="visit">
                 <div className="map">
                     <iframe title="gmaps-iframe"
-                            src={`https://maps.google.com/maps?q=${monument.address ? monument.address : monument.coordinatePointAsString}&z=16&output=embed`}
+                            src={`https://maps.google.com/maps?q=${monument.address ? monument.address : monument.lat + ',' + monument.lon}&z=16&output=embed`}
```

Comment:
This is to fix an issue with the map not displaying properly after we removed `coordinatePointAsString` from Monument.